src/plugins/maxlength/main.rs

```diff
@@ -0,0 +1,42 @@
+use overrider::*;
+
+use crate::drw::Drw;
+use crate::item::Item;
+use crate::result::*;
+use crate::clapflags::CLAP_FLAGS;
+
+#[override_flag(flag = maxlength)]
+impl Drw {
+    pub fn postprocess_matches(&mut self, mut current_matches: Vec<Item>) -> CompResult<Vec<Item>> {
+	let max_length_str_option = CLAP_FLAGS.value_of( "maxlength" );
+
+	match max_length_str_option
+	{
+		None => {
+			Err(Die::Stderr("Please specificy max length".to_owned()))
+		},
+		Some( max_length_str ) => 
+		{
+			let max_length_result = max_length_str.parse::<usize>();
+			match max_length_result
+			{
+				Err( _ ) => Err(Die::Stderr("Please specificy a positive integer for max length".to_owned())),
+				Ok( 0 ) => Ok( current_matches ),
+				Ok( max_length ) => {
+					// >= in place of = in case someoen pastes stuff in
+					// when there is a paste functionality.
+					if self.input.len() >= max_length
```

Comment:
`self.input.len()` could cause a lot of problems with non-English or other unicode characters. I've had a lot of problems with this one on the backend. How about using the following:
```rust
// at the top
use unicode_segmentation::UnicodeSegmentation;
// on line 28
self.input.graphemes(true).count()
```

unicode_segmentation is already a cargo dependency so you shouldn't need to do any dependency configuration.

Reply:
I resolved but I cannot get fcitx to work in dmenu so I have no way to test this change.